nimterop/cimport.nim

```diff
@@ -184,8 +228,8 @@ proc cSearchPath*(path: string): string {.compileTime.}=
   if result.len == 0:
     var found = false
     for inc in gStateCT.searchDirs:
-      result = (inc & "/" & path).replace("\\", "/")
-      if fileExists(result) or dirExists(result):
+      result = findPath(inc & "/" & path, fail = false)
```

Comment:
* `inc / path` (using `/` from os)
* also, need to distinguish whether path is absolute:

result = findPath(path, fail = false)
if (not path.isAbsolute) and result.len == 0: